.github/workflows/c.yaml

```diff
@@ -12,34 +12,9 @@ on:
       - "c/**"
       - .github/workflows/c.yaml
 jobs:
-  coverage:
-    name: Code Coverage
```

Comment:
Removed this stage since it isn't doing anything, we've deleted our code cov account